charts/hub-net-controller-manager/templates/azurecloudconfig.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.enableTrafficManagerFeature }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: azure-cloud-config
+  namespace: {{ .Values.fleetSystemNamespace }}
+type: Opaque
+data:
+  azure.json: {{ .Values.config.azureCloudConfig | toJson | indent 4 | b64enc | quote }}
+{{- end }}
```

Comment:
can you add a newline at the end of the json file?

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
updated